x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api.ts

```diff
@@ -184,6 +184,9 @@ export async function createAlert({
     'createdBy' | 'updatedBy' | 'muteAll' | 'mutedInstanceIds' | 'executionStatus'
   >;
 }): Promise<Alert> {
+  if (alert.actions.length === 2) {
+    alert.actions[1].group = 'resolved';
+  }
```

Comment:
Is this just for debugging purposes until the UI is ready?

Reply:
ops, yes :-)